aws/scripts/common.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
```

Comment:
This is a duplicate script.  We probably should create a top-level script directory.   Thoughts?

Reply:
this script is different from every cloud provider.